.github/workflows/bump-version/action.yml

```diff
@@ -2,25 +2,61 @@ name: Bump version for releases
 description: "Call bumpversion"
 inputs:
   part:
-    description: 'What kind of release is this?'
+    description: "What kind of release is this?"
     required: true
-    default: 'release'
+    default: "release"
 runs:
   using: "composite"
   steps:
-  - name: Set git configs for bumpversion
-    shell: bash
-    run: |
-      git config user.name 'Lance Release'
-      git config user.email 'lance-dev@lancedb.com'
-  - name: Create release version and tags
-    working-directory: python
-    shell: bash
-    run: |
-      cargo install cargo-bump
-      cargo bump ${{ inputs.part }}
-  - name: Synchronize rust version
-    shell: bash
-    run: |
-      cargo install cargo-workspaces --version 0.2.44
-      cargo ws version --no-git-commit -y --exact --force 'lance*' ${{ inputs.part }}
+    - name: Set git configs for bumpversion
+      shell: bash
+      run: |
+        git config user.name 'Lance Release'
+        git config user.email 'lance-dev@lancedb.com'
+    - name: Create release version and tags
+      working-directory: python
+      shell: bash
+      run: |
+        cargo install cargo-bump
+        cargo bump ${{ inputs.part }}
+    - name: Synchronize rust version
+      shell: bash
+      run: |
+        cargo install cargo-workspaces --version 0.2.44
+        cargo ws version --no-git-commit -y --exact --force 'lance*' ${{ inputs.part }}
+    - name: Bump java version
```

Comment:
in my previous experience, we always only auto-bump patch version. all major/minor should be manual commit. but it is up to the lance community :-P

Reply:
Yeah, lance community also only auto bump patch version, major/minor needs to be manual commit.
Lancedb release pipeline opens the door for auto bump minor version but is also not recommended.

Reply:
ok i might've misunderstood. but looksl ike the code below handles major/minor/patch bump?

Reply:
We have the capability to bump other versions, but we generally only automatically increment minor version.